flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/utils/TableSqlFunction.scala

```diff
@@ -18,39 +18,40 @@
 
 package org.apache.flink.table.functions.utils
 
-import com.google.common.base.Predicate
 import org.apache.calcite.rel.`type`.RelDataType
 import org.apache.calcite.sql._
 import org.apache.calcite.sql.`type`._
 import org.apache.calcite.sql.parser.SqlParserPos
 import org.apache.calcite.sql.validate.SqlUserDefinedTableFunction
-import org.apache.calcite.util.Util
 import org.apache.flink.api.common.typeinfo.TypeInformation
 import org.apache.flink.table.calcite.FlinkTypeFactory
-import org.apache.flink.table.functions.TableFunction
+import org.apache.flink.table.functions.{ScalarFunction, TableFunction}
 import org.apache.flink.table.plan.schema.FlinkTableFunctionImpl
 
 import scala.collection.JavaConverters._
 import java.util
 
+import org.apache.calcite.sql.`type`.SqlOperandTypeChecker.Consistency
+import org.apache.flink.api.java.typeutils.TypeExtractor
```

Comment:
I suggest that format the import order, i.e.: 
**calcite**
**calcite**
**flink.api**
**flink.api**
**flink.table**
**flink.talbe**
What do you think?